this is not a typed array.

```
~/Projects/QCU/qcu005 feature/e2e-tests* 40s
❯ npm run docs:features      

> qcu005@0.2.0 docs:features /home/zeno.jiricek/Projects/QCU/qcu005
> wintersmith build --config client/test/e2e/wintersmith.json

  building site
  using config file: /home/zeno.jiricek/Projects/QCU/qcu005/client/test/e2e/wintersmith.json
  rendering tree:
    features.css (url: /features.css)
    features.scss (url: /features.scss)
    index.md (url: /)
    membership-application.couple.feature (url: /membership-application.couple.html)
    membership-application.family.feature (url: /membership-application.family.html)
    membership-application.single.feature (url: /membership-application.single.html)
    membership-application.single.parent.feature (url: /membership-application.single.parent.html)
    product-card.feature (url: /product-card.html)

/home/zeno.jiricek/Projects/QCU/qcu005/node_modules/wintersmith-sassify/plugin.coffee:84
          return callback(null, Buffer.from(this.config.data));
                                       ^

TypeError: this is not a typed array.
  at Function.from (native)
  at Sass.<anonymous> (/home/zeno.jiricek/Projects/QCU/qcu005/node_modules/wintersmith-sassify/plugin.coffee:71:29)
  at /home/zeno.jiricek/Projects/QCU/qcu005/node_modules/wintersmith/lib/core/renderer.js:44:19
  at nextTickCallbackWith0Args (node.js:420:9)
  at process._tickCallback (node.js:349:13)


npm ERR! Linux 4.4.13-200.fc22.x86_64
npm ERR! argv "/home/zeno.jiricek/.nvm/versions/node/v4.4.7/bin/node" "/home/zeno.jiricek/.nvm/versions/node/v4.4.7/bin/npm" "run" "docs:features"
npm ERR! node v4.4.7
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE
npm ERR! qcu005@0.2.0 docs:features: `wintersmith build --config client/test/e2e/wintersmith.json`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the qcu005@0.2.0 docs:features script 'wintersmith build --config client/test/e2e/wintersmith.json'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the qcu005 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     wintersmith build --config client/test/e2e/wintersmith.json
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs qcu005
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls qcu005
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/zeno.jiricek/Projects/QCU/qcu005/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -20,6 +20,9 @@
     "async": "2.x",
     "vows": "0.8.x"
   },
+  "engines":  {
+    "node" : ">=6.0"
+  },
   "peerDependencies": {
     "wintersmith": "2.x"
   },

